test/evaluation/inputDescriptorFilterEvaluationHandler.spec.ts

```diff
@@ -112,13 +145,20 @@ describe('inputDescriptorFilterEvaluationHandler tests', () => {
     message2.payload = {"result": {"path": ["$", "issuer"], "value": "did:foo:123"}, "valid": false};
     const evaluationClient: EvaluationClient = new EvaluationClient();
     const evaluationHandler: EvaluationHandler = new InputDescriptorFilterEvaluationHandler(evaluationClient);
-    evaluationHandler.handle(presentationDefinition, new VP(presentation));
+    evaluationHandler.handle(presentationDefinition, presentation);
     expect(evaluationClient.results).toEqual([message0, message1, message2]);
   });
 
   it(`input descriptor's constraint.fields.filter match`, () => {
     const inputCandidates: unknown = getFile('./test/dif_pe_examples/vp/vp_general.json');
-    const presentation: Presentation = new Presentation(inputCandidates['@context'], inputCandidates['presentation_submission'], inputCandidates['type'], inputCandidates['verifiableCredential'], inputCandidates['holder'], inputCandidates['proof']);
+    const presentation: VerifiablePresentation = {
+      '@context': inputCandidates['@context'],
+      presentationSubmission: inputCandidates['presentation_submission'],
+      type: inputCandidates['type'],
+      verifiableCredential: inputCandidates['verifiableCredential'],
+      holder: inputCandidates['holder'],
+      proof: inputCandidates['proof']
+    };
     const presentationDefinition: PresentationDefinition = getFile('./test/resources/pd_input_descriptor_filter.json')['presentation_definition'];
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.